app/models/imported_clients_csv.rb

```diff
@@ -92,6 +92,7 @@ def import(agency)
           sixty_two_plus: yes_no_to_bool(row[SIXTY_TWO]),
```

Comment:
I'm not sure, the age is technically for HoH, which _should_ be the client in question.  I think I'll leave these as is since they are explicit questions, but use age over them when calculating a fake DOB.  That will capture the responses as uploaded for these age related questions, but will use the provided age for the matching.